libpod/options.go

```diff
@@ -2318,3 +2318,15 @@ func WithPodSlirp4netns(networkOptions map[string][]string) PodCreateOption {
 		return nil
 	}
 }
+
+// WithVolatile sets the volatile flag for the container storage.
```

Comment:
Can you add a warning here that this is potentially dangerous to use and can result in data loss?

Reply:
sure